pkgs/development/lisp-modules/builder.lisp

```diff
@@ -0,0 +1,14 @@
+(in-package :cl-user)
+(load "@asdf@")
+
+(defun load-systems ()
+  (handler-case
+      (let ((systems (uiop:getenv "systems")))
```

Comment:
Uhm, isn't the variable name way too generic? 

(I am not even sure anymore if my attempts to use variables for lists were unfixably misguided, and the proper way is passing paths to files containing the lists… although maybe for command-line overrides there are benefits for this, too)

Reply:
(unless this is strictly for in-Nix-build use)

Reply:
Yes it is only for the nix-build